app/apis/js/startup.js

```diff
@@ -4,7 +4,11 @@ var c = new Server('events', '1.0', {
   getAirplaneMode: getAirplaneMode
 });
 
+var logicAPI = new Client('logic');
+
```

Comment:
I think this is cool to use the logicAPI here. The tricky part for you will be that since this particular code will not be unloaded while the app is in background (it needs to stays), you will need to restart the iframe that contains the logicAPI when needed.
That's a cool thing though!